pkgs/development/python-modules/web_cache/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchPypi, buildPythonPackage, isPy3k }:
+
+buildPythonPackage rec {
+  pname = "web_cache";
+  version = "1.0.2";
+  name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.